examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
```

Comment:
The long term plan is somewhat hazy.  The idea is to put these in the gallery under a separate heading and then change the presentation in the gallery once the overall structure becomes clearer.